Documentation/ReleaseNotes/5.7.md

```diff
@@ -0,0 +1,18 @@
+# SwiftPM 5.7 Release Notes
+
+
+### Package Registry Support
+
+SwiftPM now supports package registry related capabilities introduced by [SE-0292](https://github.com/apple/swift-evolution/blob/main/proposals/0292-package-registry-service.md) and the corresponding [service specification](../Registry.md). With the exception of package publishing, SwiftPM can resolve and download dependencies from any compliant registry using the defined APIs. 
+
+To get started, users will need to specify their package registry by running the `swift package-registry set` subcommand or editing the `registries.json` configuration file. `swift package` tool's `--use-registry-identity-for-scm` and `--replace-scm-with-registry` options might also be of interest.
+  
+### Module Aliasing For Disambiguation
+
+Modules with the same name from different packages can now be disambiguated by module aliasing [SE-0339](https://github.com/apple/swift-evolution/blob/main/proposals/0339-module-aliasing-for-disambiguation.md). When adding a product dependency for a target in a package manifest, use a new parameter `moduleAliases` to provide a new unique name for a conflicting module. 
+
+
+* [#4119] 
```

Comment:
Should we retitle this?

Reply:
What about this section? Do we need to use a different title? All the other ones have proper titles.